docs/_future/deploy_your_app.md

```diff
@@ -7,10 +7,256 @@ layout: tutorial
 permalink: /future/deploy-your-app
 ---
 
-# Deploy your app <span class="label-beta">BETA</span>
+# Deploy your app<span class="label-beta">BETA</span>
 
 <div class="section-content">
-Instructions for deploying your next-generation Bolt application to third-party infrastructure are coming soon! Stay tuned.
+Currently, Bolt applications cannot be deployed on Slack infrastructure. However, never fear - you can deploy your Bolt app to [the Heroku platform](https://dashboard.heroku.com/) to keep your app running at all hours of the day, not just during your development sessions!
+
+In this guide, you will find the steps needed to prepare and deploy your app to Heroku, as well as how to inspect your activity logs and make updates to your app.
 </div>
 
+---
+
+### Before you begin {#before-you-begin}
+
+Before you can deploy a Bolt app to Heroku, you'll need a working Bolt app. If you haven't created one yet, [go ahead and make one](/bolt-js/future/getting-started#create-app)! If you already have an app and want features of the next-generation platform, check out the [Bolt for JavaScript migration guide](/bolt-js/future/migrate-existing-app).
+
+To test that your deploy is successful, having a Trigger that is easily invoked will be handy! Check out [the different types of Triggers](/bolt-js/future/triggers#types) and [the different ways to create a Trigger](/bolt-js/future/triggers#create) for tapping into your Workflows, creating one if necessary.
```

Comment:
I'm a fan of both the rearrangement and the tooltip! I shaved off the line about automatic workspace installation since I think readers will have either already done this or discover this for themselves in the process of creating a trigger, but I have no problem having it in to make sure there are no cracks to slip through!